app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="login-form">
+  <h2>Log in</h2>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %><br />
+      <%= f.password_field :password, autocomplete: "current-password" %>
+    </div>
+
+    <% if devise_mapping.rememberable? %>
+      <div class="field">
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
+      </div>
+    <% end %>
+
+    <div class="actions">
+      <%= f.submit "Log in" %>
+    </div>
+  <% end %>
+    
+  <%= render "devise/shared/links" %>
+
+</div>
```

Comment:
- [ ] Please modified your login view style to match the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md)

![blog_login](https://user-images.githubusercontent.com/55596970/215870491-1488912e-a2ef-4f7e-8dba-dd54785be9b0.png)